SPECS/symcrypt/symcrypt.spec

```
@@ -0,0 +1,56 @@
+%define debug_package %{nil}
+Summary:        A core cryptographic library written by Microsoft
+Name:           symcrypt
+Version:        100.20
+Release:        1%{?dist}
+License:        MIT License
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Libraries
+URL:            https://github.com/microsoft/SymCrypt
+Source0:        %{name}-%{version}-with-submodules.tar.gz
+Patch0:         build-only-generic.patch
+Patch1:         tests-build-only-generic.patch
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  python3
+BuildRequires:  python3-pyelftools
+
+%description
+A core cryptographic library written by Microsoft
+
+%prep
+%setup -n SymCrypt
+
+%patch0
+%patch1
```

Suggestion: using [`%autosetup`](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_autosetup) takes care of patching as well.

```suggestion
%autosetup -n SymCrypt
```